test/tests/functional/pbs_hook_set_jobenv.py

```diff
@@ -222,8 +221,7 @@ def common_log_match(self, daemon):
                         break
                 else:
                     self.logger.info(nomatch_msg + msg +
-                                     ' attempt ' + str(attempt))
-                attempt += 1
+                                     ' attempt ' + str(attempt+1))
```

Comment:
If you want attempt to start at one, do range(1, 60).  I didn't catch before that you started attempt at 1.  As a note, this will only have 59 attempts.

Reply:
@Bhroam, Fixed the for loop to have 60 attempts, starting from 1. Please take a look.